Embedded .NET Engine Samples/Advanced/Semaphore/App.config

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<configSections>
+		<section name="WindwardReports" type="System.Configuration.NameValueSectionHandler"/>
+	</configSections>
+	<appSettings>
+		<add key="NumberThreads" value="4"/>
+	</appSettings>
+	<startup>
```

Comment:
I think we still want to keep the <WindwardReports> section by default it in case they want to specify or play around with properties. I think removing the <WindwardReports> section may make them think to put props in the <appSettings> section. I think for the license we just want the value to be blank like: <add key="license" value=""/>.

Everything else is looking solid!